themes/edx.org/lms/templates/footer.html

```diff
@@ -1,3 +1,6 @@
+<!--Adding bootstrap  temporarily-->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```

Comment:
I believe if you added this line, it will add bootstrap to every LMS page. This is undesired. So please remove.